en-US/win10/samples/HidInjection.md

```diff
@@ -0,0 +1,78 @@
+permalink: /en-US/win10/samples/HidInjection.htm
+lang: en-US
+---
+{% include VerifiedVersion.md %}
+
+# HID Injection
+Input injection is needed for many reasons. One of the most requested reasons is to support SPI screens, which have capacitive or resistive touch panels which can be read via I<sup>2</sup>C. 
+How do you translate this touch input from the display panel to something Windows can consume?    
+
+Windows 10 IoT Core does not contain the traditional Win32 API [SendInput](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646310(v=vs.85).aspx). SendInput it is tightly bound to the legacy input system, which is not present in the image. 
+IoT Core does not currently have a replacement for this API, but we hope to introduce a compatible API in the future. In the meantime, this sample will allow you to deploy a driver to perform low level injection of touch, keyboard and
+ mouse events, and can be used until the SendInput equivelent API is available.
+
+The HID Injection sample leverages the [Virtual HID Framework](https://msdn.microsoft.com/en-us/library/windows/hardware/dn925056(v=vs.85).aspx). 
+
+# Prerequisites 
+In order to build this driver you will need the following:
+
+  * [Visual Studio 2015 Update 2](http://go.microsoft.com/fwlink/?LinkId=691129) or later.
+  * [Windows 10 Driver Development Kit](http://go.microsoft.com/fwlink/p/?LinkId=526733).
+  * [Windows 10 Assessment and Deployment Kit](https://msdn.microsoft.com/en-us/windows/hardware/dn913721(v=vs8.5).aspx#winADK).
+  * Download the [ms-iot Samples repository](https://github.com/ms-iot/samples/archive/develop.zip) from GitHub, then expand it.
+  * [Adafruit USB to TTL Cable](https://www.adafruit.com/products/954) for Kernel Debugging (if you made modifications).
+
+Optional:
+
+  * If you have git or the github app installed, you can clone the repository as well.
+
+## Building and Deploying the HID Injector
+  1. Be sure you've installed the Visual Studio update and the Windows 10 Driver Development kit before continuing.
+  1. On your development machine, use Windows Explorer to navigate to the folder where you downloaded or cloned samples.
+  1. Open the project located ```Samples\HidInjector\HidInjector.sln```.
+  1. Select the architecture you intend to deploy to: ARM for Raspberry Pi or Dragon Board, x86 for MinnowBoard Max.
+  1. You can now build the solution.
+
+### Build the Update package
+  1. Open the ```Deployment and Imaging Tools Environment``` from the Start Menu with administrative privledges (Search for the program, then right click on it and select ```Run As Administrator```)
+  1. In the command window, cd into your project directory.
+  1. cd into the driver directory.
+  1. run the script ```CreateDriverPackage.cmd```
+  
+### Copying the HID Injector to your device
+  1. Open a network share on your IoT Core device by opening the Run dialog (Win-R), then entering \\```IP for your IoT Core device\c$```. Enter credentials if prompted.
```

Comment:
I think markdown gets confused with the double slash (at least, I see only one slash in the rich preview here in GitHub).